recipes/cairo/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "1.17.2":
+    sha256: 
+    url: "https://www.cairographics.org/snapshots/cairo-1.17.2.tar.xz"
```

Comment:
It looks like snapshots are unstable, in-progress.
See header of https://cairographics.org/snapshots/

Are we ok with this?